eisbuk-admin/public/locales/en/translation.json

```diff
@@ -84,10 +89,10 @@
     "NonReversible": "This action is not reversible, the user will definitely be deleted."
   },
   "Categories": {
-    "course": "course",
-    "competitive": "competitive",
-    "pre-competitive": "pre-competitive",
-    "adults": "adults"
+    "course": "Course",
+    "competitive": "Competitive",
+    "pre-competitive": "Pre-competitive",
+    "adults": "Adults"
   },
```

Comment:
This is purposely lowercased, if you need uppercased versions, you can use `@/utils/capitalizeFirst` directly in the component, it will capitalize first letter as well as every first letter of words divided by`-`